packages/web-components/src/components/cbp-toggle/cbp-toggle.stories.tsx

```diff
@@ -0,0 +1,137 @@
+export default {
+    title: 'Components/Toggle',
+    //tags: ['autodocs'],
+    argTypes: {
+        hideStatus: {
+            description: 'Determines if the status text for the `on` and `off` is visible for the toggle control',
+            control: 'boolean'
+        },
+        disabled: {
+            description: 'Sets the disable state for the toggle control',
+            control: 'boolean'
+        },    
+        statusTexton: {
```

Comment:
Sorry, this is nitpicky, but please make the On/Off control names camelcase to be consistent with the prop names.

Reply:
update as requested